src/components/Dropdown/components/UserLogout.tsx

```diff
@@ -0,0 +1,39 @@
+import { useDispatch } from "react-redux";
+import { useLogoutUserMutation } from "@/src/services/logoutApi";
+
+import { setIsLoggedIn, setUserData } from "@/src/store/global";
+
+import { notifyError, notifySuccess } from "@/src/utils/toast";
+
+import styles from "../dropdown.module.css";
+
+export function UserLogout() {
+  const reduxDispatch = useDispatch();
+  const [logoutUser] = useLogoutUserMutation();
+
+  const logout = () => {
+    logoutUser()
+      .unwrap()
+      .then(() => {
+        reduxDispatch(setIsLoggedIn({ isLoggedIn: false }));
+        reduxDispatch(
+          setUserData({
+            first_name: null,
+            imageURL: null,
+            roles: null,
+          })
+        );
+        notifySuccess("User logged out successfully");
+      })
+      .catch((error) => {
+        const errorMessage = "Something went wrong!";
+        notifyError(errorMessage);
+      });
+  };
+
+  return (
+    <span role="button" className={styles.signout_button} onClick={logout}>
+      Sign out
+    </span>
```

Comment:
- please use a button element instead of span and role=button

Reply:
`text-align: left` will align perfectly.

Reply:
Already tried and it’s working as expected

Reply:
I meant adding `text-align: left` on the class when using button will fix the problem you had trying to use button element.

Reply:
I have tried using button then only i have made this change even in status site they are using span

Reply:
yes it is horizontally aligned but if i try using button tag to align with above element then it is not working

Reply:
Resolving this for now please create a ticket for this and add it in pr description, let's promote using semantic tags.

Reply:
sure will create ticket for it